youtrack_sdk/entities.py

```diff
@@ -154,8 +154,10 @@ class TextIssueCustomField(IssueCustomField):
 
 class SimpleIssueCustomField(IssueCustomField):
     type: Literal["SimpleIssueCustomField"] = Field(alias="$type", default="SimpleIssueCustomField")
-    value: Optional[AwareDatetime | StrictStr | StrictInt | StrictFloat] = None
-    validate_value = field_validator("value")(validate_youtrack_datetime)
+    value: Annotated[
```

Comment:
So they have implemented https://github.com/pydantic/pydantic/issues/6732 ? If yes, that's great. Please don't forget to revisit issues we open in third-party projects.

Reply:
Yes, it is implemented, `ValidationInfo` now contains `.data`.
Seems you already left a comment, should I close the pydantic issue?

Reply:
I have pinged them, and they closed it already.